src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.ts

```diff
@@ -164,6 +164,9 @@ export class AssessmentDashboardComponent implements OnInit, AfterViewInit {
             const [finishedExercises, unfinishedExercises] = partition(
                 exercises,
                 (exercise) =>
+                    exercise.numberOfAssessmentsOfCorrectionRounds
```

Comment:
a nice comment explaining the mapping would be nice :)

Reply:
We'll add that!